client/client-multi/client-cpp/impl/security/KeyUtils.cpp

```diff
@@ -35,8 +40,9 @@ SessionKey KeyUtils::generateSessionKey(std::size_t length)
 bool KeyUtils::checkKeyPair(const KeyPair &keys)
 {
     try {
-        auto &pub = keys.getPublicKey();
-        std::unique_ptr<Botan::Public_Key> ppub{Botan::X509::load_key(pub)};
+        const PublicKey& pub = keys.getPublicKey();
+        Botan::DataSource_Memory dataSrc(pub);
+        std::unique_ptr<Botan::Public_Key> ppub{Botan::X509::load_key(dataSrc)};
```

Comment:
There are no significant changes and the above code compiled well with MSVC 2013 before these changes.